tests/plugins/font/font.js

```diff
@@ -170,6 +172,43 @@
 				'<p>x<span style="font-size:12px"><em>foo</em></span><em><span style="font-size:24px">bar</span></em>x@</p>' );
 		},
 
+		// #584
```

Comment:
I see there is on in a `format` plugin. Still those are separate plugin with different code so it should be tested for both.